src/cpp/include/openvino/genai/speech_generation/text2speech_pipeline.hpp

```diff
@@ -0,0 +1,84 @@
+// Copyright (C) 2023-2025 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "openvino/genai/generation_config.hpp"
+#include "openvino/genai/speech_generation/speech_generation_config.hpp"
+#include "openvino/genai/speech_generation/speech_generation_perf_metrics.hpp"
+
+namespace ov {
+namespace genai {
+class Text2SpeechPipelineImpl;
+
+struct Text2SpeechDecodedResults {
+    std::vector<ov::Tensor> speeches;
+    SpeechGenerationPerfMetrics perf_metrics;
+};
+
+/**
+ * Text to speech pipelines which provides unified API to all supported models types.
+ */
+class OPENVINO_GENAI_EXPORTS Text2SpeechPipeline {
+public:
+    /**
+     * Initializes text to speech pipelines from a folder with models and performs compilation after it
+     * @param models_path A models path to read models and config files from
+     * @param device A single device used for all models
+     * @param properties Properties to pass to 'compile_model' or other pipeline properties like LoRA adapters
+     */
+    Text2SpeechPipeline(const std::filesystem::path& models_path,
+                        const std::string& device,
+                        const ov::AnyMap& properties = {});
+
+    template <typename... Properties,
+              typename std::enable_if<ov::util::StringAny<Properties...>::value, bool>::type = true>
+    Text2SpeechPipeline(const std::filesystem::path& models_path, const std::string& device, Properties&&... properties)
+        : Text2SpeechPipeline(models_path, device, ov::AnyMap{std::forward<Properties>(properties)...}) {}
+
+    /**
+     * Generates speeches based on input texts
+     * @param text input text for which to generate speech
+     * @param speaker_embedding  that is a vector representing the unique characteristics of a speaker's voice
+     * @param properties Speech generation parameters specified as properties
+     * @returns raw audios of the input texts spoken in the specified speaker's voice
+     */
+    Text2SpeechDecodedResults generate(const std::string& text,
+                                       const ov::Tensor& speaker_embedding = ov::Tensor(ov::element::f32, Shape{0}),
```

Comment:
```suggestion
                                       const ov::Tensor& speaker_embedding = ov::Tensor(),
```
same below

Reply:
Describe what the default value corresponds to in docstrings

Reply:
Done